cypress/integration/plugins/reports-dashboards/04-download.spec.js

```diff
@@ -137,6 +131,6 @@ describe('Cypress', () => {
 
     cy.get('#generateReportFromDetailsFileFormat').click({ force: true });
 
-    cy.get('#downloadInProgressLoadingModal');
+    cy.get('.euiToastHeader__title').contains('Successfully generated report').should('exist');
```

Comment:
I think you can utilize the id `generateReportSuccessToast` in case there are multiple toasts.